README.md

```diff
@@ -23,62 +23,72 @@
   </a>
 </p>
 
-Planktos enables websites to serve their static content over BitTorrent by turning users into seeders. This allows website owners to significantly reduce hosting costs for static content and scale in realtime without provisioning more web servers. Planktos works in vanilla Chrome and Firefox (no browser extensions needed), using [WebTorrent](https://webtorrent.io) for peer to peer file transfers and service workers to reroute network requests over BitTorrent.
+Planktos enables websites to serve their static content over BitTorrent by turning users into seeders. That means that users viewing a website with Planktos are also serving the website to other users. This allows website owners to significantly reduce hosting costs for static content, scale in real-time without provisioning more web servers, and prevent user impact during an outage. Planktos works in vanilla Chrome and Firefox (no browser extensions needed).
 
-Installing Planktos into a website is as simple as including the Planktos install.js script and using the Planktos command line interface to bundle your static files into a torrent.
+Installing Planktos into a website is as simple as including the Planktos `install.js` script and using the Planktos command line interface to bundle the website's static files into a torrent. For typical use cases, Planktos is designed to work out of the box, and for more specialized use cases, Planktos has a very simple interface for customization.
 
 A special thanks to the [WebTorrent](https://webtorrent.io) project, which is used extensively in Planktos.
 
+## Why
+
+- Let users load your static assets from each other, instead of hitting your web servers
+- Once a user visits your website for the first time, he/she will be able to load it even if your web servers go down
```

Comment:
It sounds like the user will be able to load the site without the server the first time they visit